push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/cache/SimpleApnsClientCache.java

```diff
@@ -1,13 +1,13 @@
 /**
  * JBoss, Home of Professional Open Source
  * Copyright Red Hat, Inc., and individual contributors.
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
- * 	http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
I guess these are html paragraph tags (and probably generated by something), are they? Does the fact that there are 3 of them mean that one is mismatched? :thinking: Or are they all mismatched (should they have closing `</p>` -- I can't really remember html, but I think it's usually obscenely forgiving with that kind of thing)?